### DISCUSSION TITLE
Renewing the student developer pack with a different school

### DISCUSSION THREAD

**Discussion by Altr0y** [community/community#145968] [GitHub Education] [answered]
### Hi everyone,

at the end of November the previous year, I applied and recieved the github student developer pack and have really enjoyed using it. It has offered me many great courses for me to follow aswell as a ton of great software I otherwise wouldn't be able to use. This year it is my first year at university and I was planning on redeeming / renewing the pack for another year, however I have ran into an issue. Last year I redeemed the pack with my then student email I was given at my high school. That email has since been deleted per the school policy. Now I have a new student email from my current university, but It appears I cannot renew my pack, since it keeps stating that I have already claimed the pack at *blank* mail. Is there a way to bypass this and formally renew the pack? 

Thanks for all of your suggestions!

**Comment by Ravindulakmina** [comment, marked as answer]
There are other ways to verify your student identity. Verify the relevant account with Eve.
Please explain what you are facing there, or visit github support.
If you find this important, mark it as answered.
